### PR TITLE
Action steps

### PR DESCRIPTION
**Pull request by BrennaManning**
[Don't merge til AFTER Annabel's PR.] Made this a part of edit student because it seemed to make sense best with how it makes sense to save and display it. I remember we discussed this before and there were pros and cons to both so happy to discuss more. I haven't implemented a friendlier way of deleting them/checking them off yet, but for now they pop up as alert boxes til you edit them to be completed. If a student has no action steps presently, no alert box appears.

### CODE REVIEW

File: public/javascripts/cohortVizController.js [hunk 1]
```diff
@@ -38,6 +40,16 @@ var cohortVizController = function($scope, $http, $state) {
...
+
+    var engagePeerList = [];
+    $scope.engagePeerData = formatPieData($scope.engagePeer, engagePeerList);
+
+    var engageAdultList = [];
+    $scope.engageAdultData = formatPieData($scope.engageAdult, engageAdultList);
+
+    var engageContentList = [];
+    $scope.engageContentData = formatPieData($scope.engageContent, engageContentList);
+
```
Comment (nmohamed) [inline public/javascripts/cohortVizController.js]: again, 
`$scope.engagePeerData = formatPieData($scope.engagePeer, []);`?

Comment (davidabrahams) [inline public/javascripts/cohortVizController.js]: I think these lines got changes somewhere else, but idk? Passing the empty list is unneccessary. I think is was changed on cohort-timefilter

File: public/javascripts/studentControllers.js [hunk 2]
```diff
@@ -156,6 +157,11 @@ var studentDataController = function($scope, $http, $state) {
           $scope.monthIndexList = createMonthList(response.data);
           $scope.dayIndexList = createDayList(response.data);
           $scope.isData = true;
+
```
Comment (davidabrahams) [inline public/javascripts/studentControllers.js]: remove these

File: public/views/content/dailyEntry.html [hunk 3]
```diff
@@ -47,10 +47,14 @@
           <option value="2">2</option>
         </select>
       </div>
+
+
```
Comment (davidabrahams) [inline public/views/content/dailyEntry.html]: remove

File: public/views/content/dailyEntry.html [hunk 4]
```diff
@@ -47,10 +47,14 @@
...
       </div>
+
+      
+
+
     </div>
 
     <div class="col-sm-6">
-
+
```
Comment (davidabrahams) [inline public/views/content/dailyEntry.html:52]: remove

File: public/views/content/editStudentInfo.html [hunk 5]
```diff
@@ -24,6 +24,16 @@
...
     </div>
 
+
+
+
+    <div class="form-group">
+        <label>Action Steps:</label>
+        <input type="text" ng-model="$parent.editStudent.actionSteps" placeholder="Input action steps" class="form-control" required autofocus />
+    </div>
+
```
Comment (davidabrahams) [inline public/views/content/editStudentInfo.html]: remove lots of whitespace

**Comment by davidabrahams** [comment]
I dealt with merge conflicts. Just like on Nora's branch, I'm 93% sure I didn't break anything, but @BrennaManning you should check. Other than a few stylistic things :bus:

**Comment by aconsilvio** [comment]
Add action steps to cohort + then 🚌

**Comment by shrutiyer** [comment]
:+1:

**Comment by davidabrahams** [comment]
Fixed #106

**Comment by BrennaManning** [comment]
Okay, note for all!

In starting to do this for cohorts I realized we haven't set up anything for the viewing of cohort data entries and it's pretty complicated to set up all of that in just one day. (ie: no view data, etc,)  The reason it isn't very straightforward is because we don't have a cohort model for each cohort to edit qualities of.  

Discussing this with the couple people left at the meeting we agreed it makes more sense for us now to get rid of cohort comments/action steps because of how long it would take to make all of the necessary changes so that we could view the results vs how useful they seemed to be.

If you want to discuss this further we can - but those changes are now a part of this PR.

**Comment by aconsilvio** [comment]
That sounds good to me- are people ready to merge this?

**Comment by davidabrahams** [comment]
I am. Let's give @nmohamed until the end of tonight to look over this.

**Comment by davidabrahams** [comment]
Okay so for the confused, this was the PR that removed the cohort entry form. Everyone signed off so it's fine that it was merged. If we have second thoughts later we can revert.

**Pull request merged**